docs/reference/tctl/namespace/register.md

```diff
@@ -0,0 +1,151 @@
+---
+id: register
+title: tctl namespace register
+sidebar_label: register
+description: How to register a Namespace using tctl.
+tags:
+  - reference
+  - tctl
+---
+
+The `tctl namespace register` command registers a [Namespace](/docs/content/what-is-a-namespace).
+
+`tctl namespace describe`
```

Comment:
should this be `tctl namespace register` ?

Reply:
_D'oh!_ Copypasta fail.